docs/overview/shaders-to-headers.rst

```diff
@@ -1,37 +1,10 @@
 
 
 Processing Shaders with Kompute
-=====================
-
-Demo / testing function to compile shaders
```

Comment:
We should keep this documentation for people that want to use similar dynamic string compilation for shaders similar to how we use on texts. Also important as this is how it makes it simple for people to get started without having to learn about header compilation (which would be the way to go for prod) but would allow for quick dev cycles. We can also add the example on how its used in the python side

Reply:
Done.